ux.symfony.com/src/Service/RecipeParser.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Service;
+
+class RecipeParser
+{
+    public function getTitle(string $content): string
+    {
+        if (preg_match('/^# (.+)$/m', $content, $matches)) {
+            return $matches[1];
+        }
```

Comment:
question: Why does the image and description getter throws an exception and so stop the execution in the `App\Twig\RecipeList::getRecipes(...)` method but not the title one ?